### PR TITLE
docs: DOC-730 added docs for stylus CLI

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds the Stlys CLI to the compatible page.
💻 [Preview](https://deploy-preview-1150--docs-spectrocloud.netlify.app/component#styluscliversions)
🎫 [DOC-730](https://spectrocloud.atlassian.net/browse/DOC-730)

CleanShot 2023-02-28 at 11 20 03

### CODE REVIEW

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1150#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1150#pullrequestreview-1318326387" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
